components/_patterns/01-atoms/05-forms/checkbox/_checkbox-item.twig

```diff
@@ -1 +1,5 @@
-<li class="form-item--checkbox__item"><label for="checkbox{{ key }}"><input id="checkbox{{ key }}" name="checkbox" type="checkbox" checked="checked"> {{ checkbox_title }}</label></li>
+<li class="form-item--checkbox__item">
+  <label for="checkbox{{ key }}">
+    <input id="checkbox{{ key }}" name="checkbox" type="checkbox" checked="checked"> {{ checkbox_title }}
```

Comment:
@evanmwillhite You've got "checkbox_title" here, and "radio_item" in the same place in the radio twig file. Which one makes more sense? Let's pick one and standardize.